tests/kernel/boot_ltp.pm

```diff
@@ -32,7 +33,12 @@ sub run {
         $self->wait_boot(ready_time => 500);
     }
 
-    $self->select_serial_terminal;
+    if (check_var('BACKEND', 'ipmi')) {
+        use_ssh_serial_console;
+    }
+    else {
+        $self->select_serial_terminal;
+    }
```

Comment:
Hm, can we improve that because `select_serial_terminal` would already select the right console on IPMI but is only missing the update of the serial var? I guess it would not even harm to just replace the call `select_console 'root-ssh'` within `select_serial_terminal` with a call to `use_ssh_serial_console` so you would not need the if/else here?